.github/workflows/linux-workflow.yaml

```diff
@@ -100,22 +100,16 @@ jobs:
       - name: Prepare Build Artifacts
         if: github.repository == 'open-goal/jak-project' && startsWith(github.ref, 'refs/tags/') && matrix.compiler == 'clang'
         run: |
-          mkdir -p ./ci-artifacts/
-          cp ./build/decompiler/decompiler ./ci-artifacts/linux-decompiler.bin
-          cp ./build/game/gk ./ci-artifacts/linux-gk.bin
-          cp ./build/goalc/goalc ./ci-artifacts/linux-goalc.bin
-          chmod +x ./ci-artifacts/*.bin
-          strip --strip-debug ./ci-artifacts/linux-decompiler.bin
-          strip --strip-debug ./ci-artifacts/linux-gk.bin
-          strip --strip-debug ./ci-artifacts/linux-goalc.bin
-          ls -l ./ci-artifacts/
+          mkdir -p ./ci-artifacts/out
+          ./scripts/shell/extract_build.sh ./ci-artifacts/out ./
+          zip -r ./ci-artifacts/opengoal.zip ./ci-artifacts/out
```

Comment:
You should use `7z` instead since it's already installed.

Reply:
good idea, I may switch to `tar.gz` because linux people are most likely to have that, and it's already installed

Reply:
If it's meant for Linux you should use `tar`